### PR TITLE
[Polish Pass] Copy updates throughout

### PR DESCRIPTION
**Pull request by zakkain**
This PR makes several small updates to copy across the Landing page, Review page, Calendar page, and error messaging.

- header on the front page:
image

- ordering of fields in error message + adding "address" to "Email" in that deep link list
image

- switching to correct dates for Vancouver
image

- "Review" -> "Review request" to match "Send request" on the next page
image

- removing "before you send it" from the title on the revie wpage
image

### CODE REVIEW

File: web/src/pages/RegistrationPage.js [hunk 1]
```diff
@@ -76,14 +76,14 @@ const labelNames = id => {
   switch (id) {
     case 'fullName':
       return <Trans>Full name</Trans>
+      case 'email':
+      return <Trans>Email address</Trans>
```
Comment (pcraig3) [inline web/src/pages/RegistrationPage.js]: Right, this is a good point. 🔭

File: web/src/pages/RegistrationPage.js [hunk 2]
```diff
@@ -106,6 +106,14 @@ const validate = values => {
...
     )
   }
+  if (!validateEmail(values.email)) {
+    errors.email = (
+      <Trans>
+      You need to provide an email address so we can send you a confirmation
+      message.
+      </Trans>
+    )
+  }
```
Comment (pcraig3) [inline web/src/pages/RegistrationPage.js]: Is this the right text for the email error message?

File: web/src/pages/ReviewPage.js [hunk 3]
```diff
@@ -44,9 +44,9 @@ class ReviewPage extends React.Component {
             ← <Trans>Go Back</Trans>
           </NavLink>
         </TopContainer>
-        <H2>
-          <Trans>Review your request before sending it</Trans>
-        </H2>
+        <H1>
+          <Trans>Review your request</Trans>
+        </H1>
```
Comment (pcraig3) [inline web/src/pages/ReviewPage.js]: [One of Nick's commits yesterday](https://github.com/cds-snc/ircc-rescheduler/pull/170) made this header smaller.  Are we sure this is the right thing?

**Comment by zakkain** [comment]
gonna double check on the header size issue you flagged, then I'll be happy. thanks for that catch! 👍

**Comment by zakkain** [comment]
Update: let's keep it to H1 - and we can adjust H1 sizing in our typography updates if we need to. Right now it's too small, too close to the body size. Sorry Nick!

We're good to merge IMO! 🎉

**Pull request merged**